<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="fingerprint_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>120</width>
      <height>95</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 78, 78);
color: rgb(227, 227, 227);</string>
    </property>
    <property name="text">
     <string>출석</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outing_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>120</width>
      <height>95</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 78, 78);
color: rgb(227, 227, 227);</string>
    </property>
    <property name="text">
     <string>외출</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_fingerprint_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>120</width>
      <height>95</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 78, 78);
color: rgb(227, 227, 227);</string>
    </property>
    <property name="text">
     <string>지문 등록</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>120</width>
      <height>95</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 78, 78);
color: rgb(227, 227, 227);</string>
    </property>
    <property name="text">
     <string>문 닫기</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>611</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="fingerprint_page">
     <widget class="QLabel" name="fp_label_date">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>521</width>
        <height>101</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);   
border: 1px solid rgb(70, 70, 70);
font-size: 35px;</string>
      </property>
      <property name="text">
       <string>YYYY.mm.dd 오전 HH:MM:SS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="fp_label_now">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>135</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);   
border: 1px solid rgb(70, 70, 70);
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>현재 재실: 00명</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="fp_label_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>521</width>
        <height>101</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);   
border: 1px solid rgb(70, 70, 70);
font-size: 25px;</string>
      </property>
      <property name="text">
       <string>지문을 인식해 주세요</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="fp_pushButton_on">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>135</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>등교</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fp_pushButton_out">
      <property name="geometry">
       <rect>
        <x>175</x>
        <y>135</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>하교</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="outing_page">
     <widget class="QLabel" name="out_label_date">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>521</width>
        <height>101</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);   
border: 1px solid rgb(70, 70, 70);
font-size: 35px;</string>
      </property>
      <property name="text">
       <string>YYYY.mm.dd 오전 HH:MM:SS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="out_label_now">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>135</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);   
border: 1px solid rgb(70, 70, 70);
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>현재 재실: 00명</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="out_label_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>521</width>
        <height>101</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);   
border: 1px solid rgb(70, 70, 70);
font-size: 25px;</string>
      </property>
      <property name="text">
       <string>지문을 인식해 주세요</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="out_pushButton_eating">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>135</y>
        <width>121</width>
        <height>57</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>식사</string>
      </property>
     </widget>
     <widget class="QPushButton" name="out_pushButton_lib">
      <property name="geometry">
       <rect>
        <x>175</x>
        <y>135</y>
        <width>121</width>
        <height>57</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>도서관</string>
      </property>
     </widget>
     <widget class="QPushButton" name="out_pushButton_return">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>121</width>
        <height>57</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>복귀</string>
      </property>
     </widget>
     <widget class="QPushButton" name="out_pushButton_else">
      <property name="geometry">
       <rect>
        <x>175</x>
        <y>200</y>
        <width>121</width>
        <height>57</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>기타</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="new_fingerprint_page">
     <widget class="QPushButton" name="new_pushButton_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>180</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_back">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>180</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>290</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_0">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>290</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);
border: 0px solid rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_pushButton_ok">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>290</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>확인</string>
      </property>
     </widget>
     <widget class="QLabel" name="new_label_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>521</width>
        <height>160</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);   
border: 1px solid rgb(70, 70, 70);
font-size: 35px;</string>
      </property>
      <property name="text">
       <string>학번을 입력해주세요

2423002</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
